### PR TITLE
[ES] Update `Tournaments/MWC/2014`

### PR DESCRIPTION
**Pull request by Froidland**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/MWC/2014/es.md [hunk 1]
```diff
@@ -1,143 +1,112 @@
...
+  - MWC 4K 2014
+  - MWC 7K 2014
+  - MWC2014
+  - MWC4K 2014
+  - MWC7K 2014
 ---
 
 # osu!mania World Cup 2014
 
 ![MWC 2014 logo](img/logo.jpg)
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2014/es.md:13]: image tooltips and alt-texts arent translated in the entire file

for reference

`alt-text`

File: wiki/Tournaments/MWC/2014/es.md [hunk 2]
```diff
@@ -197,70 +326,254 @@ La **osu!mania World Cup 2014** (***MWC 2014***) es un torneo por países de osu
...
+   - Dependiendo de la cantidad de registros, la cantidad total puede cambiar.
+6. Todos los equipos exitosamente registrados serán publicados después de la fase de registros.
+   - Los capitanes serán notificados en caso de que su equipo haya sido aceptado o rechazado.
+7. Los seleccionadores de mapas no pueden participar como jugadores en este torneo.
+
+### Instrucciones de las fases 
+
+1. En la primera fase (Fase de Grupos), los equipos serán divididos en 4 grupos de 5 equipos.
+2. Todos los equipos de cada grupo se enfrentarán entre sí.
+3. El ranking de cada grupo es determinado ordenando los resultados del rendimiento de cada equipos con la siguiente prioridad:
```
Comment (agatemosu) [inline wiki/Tournaments/MWC/2014/es.md:533]:
```suggestion
3. El ranking de cada grupo es determinado ordenando los resultados del rendimiento de cada equipo con la siguiente prioridad:
```

File: wiki/Tournaments/MWC/2014/es.md [hunk 3]
```diff
@@ -197,70 +326,254 @@ La **osu!mania World Cup 2014** (***MWC 2014***) es un torneo por países de osu
...
+   - 9 en Fase de Grupos
+   - 11 en Octavos de Final y Cuartos de Final
+   - 13 en Semifinales y Finales
+
+### Instrucciones para la programación de horarios
+
+1. Cada fase se realizará en **un fin de semana**.
+2. Puede que las partidas en Fase de Grupos se sobrepongan.
+3. Los grupos de 4K y 7K serán programados separadamente.
+4. Todas las fases de eliminación se realizan ya sea un Sábado o un Domingo por grupo (ver la programación del torneo).
```
Comment (agatemosu) [inline wiki/Tournaments/MWC/2014/es.md:577]:
```suggestion
4. Todas las fases de eliminación se realizan ya sea un sábado o un domingo por grupo (ver la programación del torneo).
```

File: wiki/Tournaments/MWC/2014/es.md [hunk 4]
```diff
@@ -156,14 +125,102 @@ La **osu!mania World Cup 2014** (***MWC 2014***) es un torneo por países de osu
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: **Filipinas** (**[Tokiiwa](https://osu.ppy.sh/users/4029511)**, [Ainyan](https://osu.ppy.sh/users/3770641), [arcwinolivirus](https://osu.ppy.sh/users/2039089)) | ::{ flag=TH }:: **Tailandia** (**[lostcool](https://osu.ppy.sh/users/766374)**, [Mystical Faith](https://osu.ppy.sh/users/4280188), [Popo\[Mikoto\]](https://osu.ppy.sh/users/445236)) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=MY }:: **Malasia** (**[Gon](https://osu.ppy.sh/users/583765)**, [\_Sylveon\_](https://osu.ppy.sh/users/3936677), [LongGone](https://osu.ppy.sh/users/35882), [ZYuan](https://osu.ppy.sh/users/3337688)) | ::{ flag=MY }:: **Malasia** (**[Gon](https://osu.ppy.sh/users/583765)**, [\_Sylveon\_](https://osu.ppy.sh/users/3936677), [LongGone](https://osu.ppy.sh/users/35882)) |
```
Comment (agatemosu) [inline wiki/Tournaments/MWC/2014/es.md:145]:
```suggestion
| ![Corona de oro](/wiki/shared/crown-gold.png "1er lugar") | ::{ flag=KR }:: **Corea del Sur** (**[ideu-](https://osu.ppy.sh/users/137227)**, [Abcdullah](https://osu.ppy.sh/users/4140104), [FaL3C\_ChoBo](https://osu.ppy.sh/users/1218632), [jhlee0133](https://osu.ppy.sh/users/140148)) | ::{ flag=KR }:: **Corea del Sur** (**[ideu-](https://osu.ppy.sh/users/137227)**, [Abcdullah](https://osu.ppy.sh/users/4140104), [jhlee0133](https://osu.ppy.sh/users/140148), [WindyS](https://osu.ppy.sh/users/1190879)) |
| ![Corona de plata](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: **Filipinas** (**[Tokiiwa](https://osu.ppy.sh/users/4029511)**, [Ainyan](https://osu.ppy.sh/users/3770641), [arcwinolivirus](https://osu.ppy.sh/users/2039089)) | ::{ flag=TH }:: **Tailandia** (**[lostcool](https://osu.ppy.sh/users/766374)**, [Mystical Faith](https://osu.ppy.sh/users/4280188), [Popo\[Mikoto\]](https://osu.ppy.sh/users/445236)) |
| ![Corona de bronce](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=MY }:: **Malasia** (**[Gon](https://osu.ppy.sh/users/583765)**, [\_Sylveon\_](https://osu.ppy.sh/users/3936677), [LongGone](https://osu.ppy.sh/users/35882), [ZYuan](https://osu.ppy.sh/users/3337688)) | ::{ flag=MY }:: **Malasia** (**[Gon](https://osu.ppy.sh/users/583765)**, [\_Sylveon\_](https://osu.ppy.sh/users/3936677), [LongGone](https://osu.ppy.sh/users/35882)) |
```

File: wiki/Tournaments/MWC/2014/es.md [hunk 5]
```diff
@@ -156,14 +125,102 @@ La **osu!mania World Cup 2014** (***MWC 2014***) es un torneo por países de osu
...
-| ::{ flag=PH }:: Filipinas | ::{ flag=PL }:: Polanda | ::{ flag=PT }:: Portugal | ::{ flag=NO }:: Noruega |
+| ::{ flag=PH }:: Filipinas | ::{ flag=PL }:: Polonia | ::{ flag=PT }:: Portugal | ::{ flag=NO }:: Noruega |
 | ::{ flag=MY }:: Malasia | ::{ flag=HK }:: Hong Kong | ::{ flag=CL }:: Chile | ::{ flag=TH }:: Tailandia |
 | ::{ flag=DE }:: Alemania | ::{ flag=RU }:: Rusia | ::{ flag=JP }:: Japón | ::{ flag=GB }:: Reino Unido |
 | ::{ flag=US }:: Estados Unidos | ::{ flag=CN }:: China | ::{ flag=FR }:: Francia | ::{ flag=NL }:: Países Bajos |
 
-## Mappools 4K
+![](img/brackets-4K.jpg "MWC 2014 4K bracket")
+
+![](img/brackets-7K.jpg "MWC 2014 7K bracket")
```
Comment (agatemosu) [inline wiki/Tournaments/MWC/2014/es.md:135]:
```suggestion
![](img/brackets-4K.jpg "MWC 2014 Grupo 4K")

![](img/brackets-7K.jpg "MWC 2014 Grupo 7K")
```

File: wiki/Tournaments/MWC/2014/es.md (hunk 1 above)
Comment (agatemosu) [inline wiki/Tournaments/MWC/2014/es.md:13]:
```suggestion
![Logotipo de la MWC 2014](img/logo.jpg)
```

**Comment by TPGPL** [comment]
converting to draft since there has been no activity for months

File: wiki/Tournaments/MWC/2014/es.md [hunk 6]
```diff
@@ -1,143 +1,112 @@
...
-| Australia ::{ flag=AU }:: | ::{ flag=DE }:: Alemania | **9.00 (9am)** | 19.00 (7pm) UTC +10 | 11.00 (11am) UTC +2 |
-
-## Links
-
-- [osu!mania 2014 en Twitch](https://www.twitch.tv/osulive)
-- [Forma de Registro](https://docs.google.com/forms/d/1mNcRLbrD7dN9ZbToFiBdFqPcdW4_T8aotASGPOXJjZY)
-- [Hilo de Discusión (en inglés)](https://osu.ppy.sh/community/forums/topics/222100)
-- [Estadísticas: Grupo 4K](https://mwc.servegame.com/results/view/1)
-- [Estadísticas: Grupo 7K](https://mwc.servegame.com/results/view/2)
+| Administrador | ::{ flag=DE }:: [Deif](https://osu.ppy.sh/users/318565), ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366), ::{ flag=DE }:: [p3n](https://osu.ppy.sh/users/123703) |
```
Comment (juankristal) [inline wiki/Tournaments/MWC/2014/es.md:46]:
```suggestion
| Administrador | ::{ flag=ES }:: [Deif](https://osu.ppy.sh/users/318565), ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366), ::{ flag=DE }:: [p3n](https://osu.ppy.sh/users/123703) |
```

File: wiki/Tournaments/MWC/2014/es.md [hunk 7]
```diff
@@ -1,143 +1,112 @@
...
+| Seleccionador de mapas | ::{ flag=RU }:: [\[Shi-Ra\]](https://osu.ppy.sh/users/2008489) // ::{ flag=CL }:: [Agka](https://osu.ppy.sh/users/64811) |
+| Streamer | ::{ flag=AU }:: [Marcin](https://osu.ppy.sh/users/722665), ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2), ::{ flag=FR }:: [shARPII](https://osu.ppy.sh/users/776257) |
```
Comment (juankristal) [inline wiki/Tournaments/MWC/2014/es.md:48]:
```suggestion
| Streamer | ::{ flag=PL }:: [Marcin](https://osu.ppy.sh/users/722665), ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2), ::{ flag=FR }:: [shARPII](https://osu.ppy.sh/users/776257) |
```

File: wiki/Tournaments/MWC/2014/es.md [hunk 8]
```diff
@@ -1,143 +1,112 @@
...
+| Comentarista | ::{ flag=US }:: [\[Shi-Ra\]](https://osu.ppy.sh/users/2008489), ::{ flag=US }:: [Charles445](https://osu.ppy.sh/users/85000), ::{ flag=FR }:: [Mr Color](https://osu.ppy.sh/users/116078), ::{ flag=IT }:: [Roxas](https://osu.ppy.sh/users/1041833), ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347) |
```
Comment (juankristal) [inline wiki/Tournaments/MWC/2014/es.md:49]:
```suggestion
| Comentarista | ::{ flag=RU }:: [\[Shi-Ra\]](https://osu.ppy.sh/users/2008489), ::{ flag=US }:: [Charles445](https://osu.ppy.sh/users/85000), ::{ flag=FR }:: [Mr Color](https://osu.ppy.sh/users/116078), ::{ flag=IT }:: [Roxas](https://osu.ppy.sh/users/1041833), ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347) |
```

**Pull request closed without merging**